index.js

```diff
@@ -81,18 +52,105 @@ function generateSquares(numLynx, miss) {
     })
 }
 
-function turnCard() {
+function turnCard(numLynx, miss) {
     // Grabs all gridItem divs as gridItems to be used in forEach
     let gridItems = document.querySelectorAll('.gridItem')
     gridItems.forEach((item) => {
         item.addEventListener('click', () => {
-            if(item.dataset.hit === '1') {
+            if (item.dataset.hit === '1') {
                 item.style.backgroundImage = "url('project-assets/babylynx2.jpg')"
                 item.textContent = 'Hooray, you\'ve found a lynx!'
-            } else {
+                hits++
+                item.dataset.hit = '2'
+                if (hits === numLynx) {
+                    modalDiv.style.display = "inline-block"
```

Comment:
whole block a potential function to be called with params passed in for the unique data?